src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -12,7 +12,7 @@
     <_CortypeFilePath Condition=" '$(_CortypeFilePath)' == '' ">$(MSBuildThisFileDirectory)..\inc\cortypeinfo.h</_CortypeFilePath>
     <_RexcepFilePath Condition=" '$(_RexcepFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\rexcep.h</_RexcepFilePath>
     <_ILLinkTrimXmlFilePath Condition=" '$(_ILLinkTrimXmlFilePath)' == '' ">$(MSBuildThisFileDirectory)ILLinkTrim.xml</_ILLinkTrimXmlFilePath>
-    <_ILLinkTasksToolsDir>$(PackagesDir)/illink.tasks/$(ILLinkTasksPackageVersion)/tools</_ILLinkTasksToolsDir>
+    <_ILLinkTasksToolsDir>$(PkgILLink_Tasks)/tools</_ILLinkTasksToolsDir>
```

Comment:
@jkoritzinsky where will `PkgILLink_Tasks` be set?

Reply:
Ah, thanks. Curious that it didn't cause ci issues - I see `GeneratePackagePath` used in https://github.com/dotnet/coreclr/pull/27373/files/0e2c34c1d618bc9b63284bc2b5a6edaac8fdbc3a#diff-77f79d3dc4e982240394162f15e25ffbR82 as well. I'll look into the linker path since I'm trying to get illink updated anyway.